src/Task.js

```diff
@@ -0,0 +1,23 @@
+import "./Task.css";
+import editIcon from "./edit.png";
+import deleteIcon from "./delete.png";
+
+function Task(props) {
+    return (
+        <div className="listItem" id={props.isChecked ? "completedTask" : "incompleteTask"}>
```

Comment:
IDs should be unique within a web page. You probably want to be using classes instead.